polytracker/include/taintdag/outputfile.h

```diff
@@ -97,7 +97,7 @@ template <Section... Sections> class OutputFile {
         (SectionMeta{.tag = Sections::tag,
                      .align = Sections::align_of,
                      .offset = 0,
-                     .size = Sections::allocation_size})...};
+                     .size = 0})...};
```

Comment:
Does setting the size to 0 for a section mean the section technically won't exist when output?

Reply:
I don't think so? I guess this was done to have correct information in the case when we have a section that's optional, like `Events` if function tracing is turned off. I think there should be a header entry for the section, but it should tell you there's no trace events in it and it's size is 0? Am I right @hbrodin ?

Reply:
Also we should update the comment above this code to reflect this change :)

Reply:
This is just the initial value. The final value will be set in the destructor of the `OutputFile`. That behaviour hasn't changed, it is just that we don't assume the size to be max from the beginning, but rather to be empty.